### ISSUE TITLE
getAndAddContracts.js:14-16: Provide autocomplete feature...

### ISSUE THREAD

**Issue by 0pdd** [self-xdsd/self-web#104]
The puzzle `103-573d7d5f` from #103 has to be resolved: 

https://github.com/self-xdsd/self-web/blob/81d9eeaaaa9b1849a87811d8cff515478bac21c9/src/main/resources/public/js/getAndAddContracts.js#L14-L16

The puzzle was created by cristianpela on 21-Sep-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@amihaiemil please pay attention to this new issue.

**Comment by criske** [comment]
@amihaiemil I'll work on this

**Comment by amihaiemil** [comment]
@criske I suggest you make the first API call only after the first 3 characters and then once gor each second char.

Also, maybe the username will start with ``@``, which should of course be ignored : D

Dont forget bout PDD, you dont have to fix it all in 1h.

**Comment by criske** [comment]
@amihaiemil also adding a debounce time, maybe about 500ms, to avoid input spamming.

**Comment by amihaiemil** [comment]
Paid with transaction id 00N02142EM643632V

**Comment by 0pdd** [comment]
The puzzle `103-573d7d5f` has disappeared from the source code, that's why I closed this issue. //cc @amihaiemil
